doc/reference/common-weaknesses/size-limits.rst

```diff
@@ -0,0 +1,15 @@
+Size limits may prevent transactions succeeding, and prevent recovery of funds
```

Comment:
Again, I meant this to be subsumed by https://plutus.readthedocs.io/en/latest/reference/common-weaknesses/hard-limits.html#